static/_redirects

```diff
@@ -13,7 +13,7 @@ support/how-to/rackspace-email-password-recovery-faq/  /support/how-to/manage-pa
 
 # Retired Articles
 
-windows-server-2012-r2-with-managed-operations-support/ /how-to/article-retired/ 404
```

Comment:
@viridaria did you mean to remove this redirect, or to add your redirect in the line above it? (I assume the latter)

Reply:
add it

Reply:
ok then you'll need to redo your commit to add your redirect instead of replacing the other one, because as written it will break that existing redirect.